src/res/defaults.json

```diff
@@ -258,7 +258,12 @@
         "https://pgp.mit.edu",
         "https://keys.mailvelope.com"
       ],
-      "mvelo_tofu_lookup": true
+      "mvelo_tofu_lookup": true,
+      "wkd_lookup": true,
+      "wkd_blacklist": [
+        "gmail\\..*",
+        "gmx\\..*"
+      ]
```

Comment:
Should also include: web.de, hotmail\\..*, outlook\\..*, yahoo\\..*

Reply:
I added the ones you specified. I don't want to blacklist too much as we hope that more providers will do WKD ;-) Maybe it helps if they see that they get many queries. But in this list I would say "no chance at all" maybe except for the german providers.